File: src/main/java/io/github/dqualizer/dqlang/types/rqa/definition/loadtest/ResponseMeasures.java
package io.github.dqualizer.dqlang.types.rqa.definition.loadtest;

import com.fasterxml.jackson.annotation.JsonProperty;
import io.github.dqualizer.dqlang.types.rqa.definition.enums.ResponseTime;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ResponseMeasures {
    @JsonProperty("response_time")
    private ResponseTime responseTime;
}

File: src/main/kotlin/io/github/dqualizer/dqlang/types/rqa/definition/loadtest/ResponseMeasures.kt
package io.github.dqualizer.dqlang.types.rqa.definition.loadtest

import com.fasterxml.jackson.annotation.JsonProperty
import io.github.dqualizer.dqlang.types.rqa.definition.enums.Satisfaction

data class ResponseMeasures(
  @JsonProperty("response_time")
  val responseTime: Satisfaction? = null
)
